week-2/D-predicates/exercise.js

```diff
@@ -7,12 +7,12 @@
 
 // Finish the predicate function to test if the passed number is negative (less than zero)
 function isNegative(number) {
-
+ return number<0;
 }
 
 // Finish the predicate function to test if the passed number is between 0 and 10
 function isBetweenZeroAnd10(number) {
-
+ return 0<number<10;
```

Comment:
This passes `true` because `0 < number`. But the second part, `number < 10`, doesn't get evaluated. To evaluate two comparisons, you need to use the `&&`:

```
return number > 0 && number < 10;
```